demo/theme.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Added a theme page to showcase different components (thought something like this would be nice)

Reply:
➕

Reply:
I like this. Could you add a link to it from the index page?